triggers/event-fired/event.schema.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+      "body": {
+        "type": "string",
+        "description": "Body text of the event"
+      },
+      "last_updated": {
+        "type": "string",
+        "description": "Last time event was updated"
+      },
+      "event_type": {
+        "type": "string",
+        "description": "Type of event"
+      },
+      "title": {
+        "type": "string",
+        "description": "Title of event"
+      },
+      "date": {
+        "type": "string",
+        "description": "Date event occurred"
```

Comment:
Do we want to add "format": "date-time" here? Lets just make sure that format applies